vulnerabilities/models.py

```diff
@@ -119,6 +119,11 @@ class Meta:
         )
         ordering = ["reference_id", "url"]
 
+    def save(self, *args, **kwargs):
```

Comment:
I am not sure this is the correct way to make a field mandatory.
Rather 
1. you need a data migration first to fix things
2. the model needs to be changed: https://github.com/nexB/vulnerablecode/pull/910/files#diff-7f9f2c92e7163b06d21fa139369d75caed6561d0368b60ea2516cece0220eb5bR108 this should be explicit blank=False, null=False
?